SteamKit2/SteamKit2/Steam/CMClient.cs

```diff
@@ -233,7 +233,7 @@ public void Connect( ServerRecord? cmServer = null )
         /// </summary>
         public void Disconnect() => Disconnect( userInitiated: true ); 
 
-        void Disconnect( bool userInitiated)
+        internal void Disconnect( bool userInitiated )
```

Comment:
Does `protected` work here instead?

Reply:
It does, but then anyone extending CMClient can see it, do you actually want that?

Reply:
hmm, how about “private protected” then?